packages/common/test/interceptor/file-upload.interceptor.spec.ts

```diff
@@ -59,8 +104,29 @@ describe('FastifyFileInterceptor', () => {
     expect(nextHandler.handle).toHaveBeenCalled();
   });
 
+  it('should throw error when process.env.STORAGE_ENDPOINT is not defined or empty', async () => {
+    // mocking env vars
+    const OLD_ENV = process.env;
+    
+  });
```

Comment:
is this expected.

Reply:
seems to have been a mix up, will fix it asap